src/core/lwip/pbuf.h

```diff
@@ -117,7 +118,6 @@ typedef void (*pbuf_free_custom_fn)(struct pbuf *p);
 struct pbuf_custom {
```

Comment:
Lets get rid of pbuf_custom now that it has only one member.

Reply:
Done.